services/wallet/bridge/cbridge.go

```diff
@@ -5,9 +5,9 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io/ioutil"
 	"math/big"
-	"net/http"
+	netUrl "net/url"
```

Comment:
Why alias this import? I can't see a conflict with any other package import.

Reply:
You're absolutely right, while I was on it I had many other imports there and probably added because of some conflicts, but at the end when I cleaned up the code a bit, forgot to remove it.